consoles/node.html

```diff
@@ -23,6 +24,7 @@
   <td{{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
   <td>{{ template "prom_query_drilldown" (args (printf "100 * (1 - avg by(instance)(rate(node_cpu{job='node',mode='idle',instance='%s'}[5m])))" .Labels.instance) "%" "printf.1f") }}</td>
   <td>{{ template "prom_query_drilldown" (args (printf "node_memory_MemFree{job='node',instance='%s'} + node_memory_Cached{job='node',instance='%s'} + node_memory_Buffers{job='node',instance='%s'}" .Labels.instance .Labels.instance .Labels.instance) "B" "humanize1024") }}</td>
+  <td>{{ template "prom_query_drilldown" (args (printf "rate(node_network_transmit_bytes{job='node',instance='%s'}[5m])" .Labels.instance ) "B/s" "humanize1024")}} / {{ template "prom_query_drilldown" (args (printf "rate(node_network_receive_bytes{job='node',instance='%s'}[5m])" .Labels.instance) "B/s" "humanize1024")}} </td>
```

Comment:
In relation to this argument of a for/against aggregating the interfaces values to one value as an indicator, I think its fine to pretend that it's a simple number for the overview page. If you start getting into that fine grain detail of breaking up the number for the network interfaces you would probably have to do the same for the CPU/Memory groupings as well. e.g. individual cpu and memory usage if you are using cgroups or whatever placement policy that you might have etc... Then all of a sudden it gets messy and at that point you may as well have a specific page (as there  is now for disk and cpu).